routes/complex.js

```diff
@@ -692,6 +692,7 @@ var complex = function (coverage) {
   });
 
   router.get('/readonlyproperty/valid', function (req, res, next) {
+    coverage['getComplexReadOnlyPropertyValid']++;
```

Comment:
You need to pre-declare it to 0:
https://github.com/Azure/autorest.testserver/blob/1726c2de57238c1c359779fd9f7335c890c9c35b/app.js#L118

Reply:
Done.